### PR TITLE
Stars can have FD bonus, add it, make it displayed

### PR DESCRIPTION
**Pull request by Eahlstan**

### CODE REVIEW

File: EliteDangerous/JournalEvents/JournalScan_Display.cs [hunk 1]
```diff
@@ -436,7 +436,14 @@ public string MassEMText()
...
                 {
-                    information.Append(' ').Append(ev.EstimatedValueFirstMappedEfficiently.ToString("N0")).Append(" cr");
+                    information.Append(' ').Append((ev.EstimatedValueFirstMappedEfficiently > 0 ? ev.EstimatedValueFirstMappedEfficiently : ev.EstimatedValueBase).ToString("N0")).Append(" cr");
+                }
+            }
+            else if (js.WasDiscovered == false && js.IsStar)
+            {
+                if (showvalues)
+                {
+                    information.Append(' ').Append(ev.EstimatedValueFirstDiscovered.ToString("N0")).Append(" cr");
```
Comment (robbyxp1) [inline EliteDangerous/JournalEvents/JournalScan_Display.cs:446]: So, i know this is a bit pedantic, but for scans before 3.2, this would show zero, as FirstDiscovered would not be set (see EstimatedValues). I suggest a test in which variable you pick, like 439.

File: EliteDangerous/JournalEvents/JournalScan_Display.cs [hunk 2]
```diff
@@ -436,7 +436,14 @@ public string MassEMText()
                 information.Append(" (Discovered)".T(EDCTx.JournalScanInfo_DIS));
                 if (showvalues)
                 {
-                    information.Append(' ').Append(ev.EstimatedValueFirstMappedEfficiently.ToString("N0")).Append(" cr");
+                    information.Append(' ').Append((ev.EstimatedValueFirstMappedEfficiently > 0 ? ev.EstimatedValueFirstMappedEfficiently : ev.EstimatedValueBase).ToString("N0")).Append(" cr");
```
Comment (robbyxp1) [inline EliteDangerous/JournalEvents/JournalScan_Display.cs:439]: Why did you add this? is this for pre-3.2 scans?

Comment (Eahlstan) [reply EliteDangerous/JournalEvents/JournalScan_Display.cs:439]: Discovered stars are never mapped, so this would query FME for all discovered stars, showing 0 as result. So it's either this to fall back to the correct base value for discovered stars or change the condition to
js.WasDiscovered == true && js.WasMapped == false && js.IsPlanet
and add a second condition
js.WasDiscovered == true && js.WasMapped == false && js.IsStar

File: EliteDangerous/JournalEvents/JournalScan_Display.cs (hunk 1 above)
Comment (Eahlstan) [reply EliteDangerous/JournalEvents/JournalScan_Display.cs]: Fixed, thanks for reminding me of ancient data.
